CMakeLists.txt

```diff
@@ -0,0 +1,204 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8.13)
+
+# Configuration options
+option(vcdiff_build_tests "Build all of vcdiff tests." ON)
+option(vcdiff_build_exec "Build vcdiff exectuable" ON)
+
+# Add subprojects first so we won't interfere with config.h.in
+
+IF (vcdiff_build_exec)
+  SET(GFLAGS_BUILD_gflags_LIB OFF)
+  SET(GFLAGS_BUILD_gflags_nothreads_LIB ON)
+  SET(GFLAGS_INSTALL_HEADERS OFF)
+  SET(GFLAGS_INSTALL_SHARED_LIBS OFF)
+  SET(GFLAGS_INSTALL_STATIC_LIBS OFF)
+  ADD_SUBDIRECTORY(gflags)
+ENDIF()
+
+IF (vcdiff_build_tests)
+  SET(gtest_install OFF CACHE BOOL "Don't install gtest" FORCE)
+  ADD_SUBDIRECTORY(googletest/googletest)
+ENDIF()
+
+PROJECT(open-vcdiff CXX C)
+SET(PROJECT_VERSION 1.0)
+SET(PROJECT_SOVERSION 1)
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -O2")
+
+INCLUDE(CheckIncludeFiles)
+CHECK_INCLUDE_FILES(ext/rope HAVE_EXT_ROPE)
+CHECK_INCLUDE_FILES(getopt.h HAVE_GETOPT_H)
+CHECK_INCLUDE_FILES(malloc.h HAVE_MALLOC_H)
+CHECK_INCLUDE_FILES(sys/mman.h HAVE_SYS_MMAN_H)
+CHECK_INCLUDE_FILES(sys/time.h HAVE_SYS_TIME_H)
+CHECK_INCLUDE_FILES(unistd.h HAVE_UNISTD_H)
+CHECK_INCLUDE_FILES(windows.h HAVE_WINDOWS_H)
+
+INCLUDE(CheckFunctionExists)
+CHECK_FUNCTION_EXISTS(gettimeofday HAVE_GETTIMEOFDAY)
+CHECK_FUNCTION_EXISTS(QueryPerformanceCounter HAVE_QUERYPERFORMANCECOUNTER)
```

Comment:
I am not sure why but this does not work.
Since this is a Windows function anyway, could you remove that line and change the one reference to `HAVE_WINDOWS_H`? It is in `src/config.h`